### PR TITLE
[Do not merge] Updating autorest for model validator plugin

### PR DESCRIPTION
**Pull request by veronicagg**
Change to plugin the model validator into autorest. 
Should not get merged until https://github.com/Azure/oav/pull/170 gets merged, as we need to republish oav.

### CODE REVIEW

File: src/autorest-core/resources/default-configuration.md [hunk 1]
```diff
@@ -70,6 +70,11 @@ use-extension:
   "@microsoft.azure/autorest.typescript": "~2.0.0"
 ```
 
+``` yaml $(model-validator)
```
Comment (fearthecowboy) [inline src/autorest-core/resources/default-configuration.md]: That infers that in order to enable it people should say `--model-validator` on the command line.

👍

Comment (veronicagg) [reply src/autorest-core/resources/default-configuration.md]: yes, or model-validator true in their readme.md file, are there any docs that should be updated to indicate this?

**Comment by olydis** [comment]
# publish job
~~~ Haskell
> waiting for PR to get merged or closed
~~~

**Comment by olydis** [comment]
# publish job
~~~ Haskell
> waiting for PR to get merged or closed
~~~

**Comment by olydis** [comment]
# AutoRest automatic publish job
~~~ Haskell
> will publish once/if PR gets merged
~~~

**Comment by olydis** [comment]
# 🤖 AutoRest automatic publish job 🤖
~~~ Haskell
> will publish once PR gets merged
~~~

**Comment by olydis** [comment]
# 🤖 AutoRest automatic publish job 🤖
~~~ Haskell
> will publish once PR gets merged
~~~

**Comment by olydis** [comment]
# 🤖 AutoRest automatic publish job 🤖
~~~ Haskell
> will publish once PR gets merged
~~~

**Comment by veronicagg** [comment]
~~~
> linux-x64 restart
~~~

**Comment by veronicagg** [comment]
~~~
> win32-x64 restart
~~~

**Pull request merged**